instrumentation/logger/lib/opentelemetry/instrumentation/logger/patches/logger.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module Logger
+      module Patches
+        # Instrumention for methods from Ruby's Logger class
+        module Logger
+          # TODO: Make sure OTel logs aren't instrumented
+          # TODO: How to pass attributes?
+          def format_message(severity, datetime, progname, msg)
+            formatted_message = super(severity, datetime, progname, msg)
+            return formatted_message if skip_instrumenting?
+
+            OpenTelemetry.logger_provider.logger(
+              name: OpenTelemetry::Instrumentation::Logger::NAME,
+              version: OpenTelemetry::Instrumentation::Logger::VERSION
+            ).on_emit(
+              severity_text: severity,
+              severity_number: severity_number(severity),
+              timestamp: datetime,
+              body: msg # New Relic uses formatted_message here. This also helps us with not recording progname, because it is included in the formatted message by default. Which seems more appropriate?
```

Comment:
Formatted message is what you're used to seeing in stdout or your logs. By default, it includes severity, timestamp, process.pid, and progname in addition to the message.

I don't see a clear answer from the [Logs Data Model entry on `body`](https://opentelemetry.io/docs/specs/otel/logs/data-model/#field-body).